pyproject.toml

```diff
@@ -11,18 +11,19 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.6.2"
-nautobot = "^1.1.0"
+# nautobot = "^1.1.5"
+nautobot = { git = "https://github.com/nautobot/nautobot.git", branch = "next" }
 # Package version detection in older versions of Python
-importlib-metadata = {version = "~3.10.0", python = "<3.8"}
+importlib-metadata = {version = "~4.4", python = "<3.10"}
 # Rendering of markdown files to HTML
-markdown = "3.3.4"
+markdown = "~3.3.6"
```

Comment:
I think these should either be implicit dependencies inherited from nautobot, or at worst `'*'` dependencies - otherwise we risk a mismatch between the versions this plugin requires and the versions that Nautobot core requires.

Reply:
Yeah. I did that for dev. dependencies. Let me give this a shot.

Reply:
Yep it worked.